packages/docusaurus-theme-redoc/src/theme/useSpec/index.ts

```diff
@@ -0,0 +1,3 @@
+import { useSpec } from './useSpec';
+
+export default useSpec;
```

Comment:
You prefer 
```
import { useSpec } from '@theme/useSpec';
```
than
```
import useSpec from '@theme/useSpec';
```
I think I was already loading 

```
import useSpecData from '@theme/useSpecData';
```

Reply:
It's less about preference and more about not breaking existing theme components if anyone is using it. So it should remain whatever it was, if you want to change it then keep both exports and add a deprecated message and then it can be removed when v3 is released

Reply:
allright - it should not add breaking change

Reply:
I don't think there is a breaking change with the `useSpec` export.

On master branch, useSpec is inside : 
[packages/ docusaurus-theme-redoc /src /utils/](https://github.com/rohit-gohri/redocusaurus/tree/main/packages/docusaurus-theme-redoc/src/utils)

This MR moves this file inside 
packages/ docusaurus-theme-redoc/ src /theme /useSpec

I don't think anyone can import a file inside `src/utils` folder, it needs to be inside the `src/theme` to be importable from outside the lib

So it's something new (no breaking change)